console/src/main/java/com/arcadedb/console/Console.java

```diff
@@ -714,7 +715,11 @@ private void checkHasSpaces(final String key, final String value) {
   }
 
   private String parseLocalUrl(final String url) {
-    return databaseDirectory + url.replaceFirst("file://", "");
+    if(url.startsWith(LOCAL_PREFIX + "//")) {
+      return url.replaceFirst(LOCAL_PREFIX + "//", "/");
```

Comment:
This forces the path tp be absolute, because it prefix the path with /. Is this wanted or a bug?

Reply:
The paths are supposed to be absolute. The intention is to not rely on the current work-directory. Any not-absolute path would always be relative to the CWD. The URL without the LOCAL_PREFIX is still supported and allows to connect to a db relative to the database directory.